storage/rocksdb/ha_rockspart.h

```diff
@@ -41,6 +41,7 @@ class ha_rockspart : public native_part::Partition_base {
   enum row_type get_partition_row_type(const dd::Table *, uint) override {
     return ROW_TYPE_NOT_USED;
   }
+  bool allow_unsafe_alter() override;
```

Comment:
Perhaps const this method and the handler as well?
```suggestion
  bool allow_unsafe_alter() const override;
```